### PR TITLE
chore: added description for Upstream module

### PR DESCRIPTION
**Pull request by guoqqqi**
Please answer these questions before submitting a pull request

**Why submit this pull request?**
- [x] New feature provided

**Related issues**
related PR： https://github.com/apache/apisix/pull/3815

### New feature or improvement
- Describe the details and related test reports.
added description for the Upstream module.
image
image

### CODE REVIEW

File: web/src/pages/Upstream/locales/zh-CN.ts [hunk 1]
```diff
@@ -37,7 +37,7 @@ export default {
   'page.upstream.step.type': '类型',
   'page.upstream.step.create.node': '创建节点',
   'page.upstream.step.pass-host': '传递域名',
-  'page.upstream.step.pass-host.tips': '当选择节点时，节点域名只能有一条数据。',
+  'page.upstream.step.pass-host.tips': '透传：将客户端的 host 透传给上游；节点：使用 upstream node 中配置的 host；重写：使用配置项 upstream_host 的值',
   'page.upstream.step.pass-host.pass': '透传',
```
Comment (juzhiyuan) [inline web/src/pages/Upstream/locales/zh-CN.ts:41]: We'd better use the original values here, like https://github.com/apache/apisix/pull/3815/files#diff-9a50e22eeadbd97e92720f01f659aa141aba58d11594757cd87e5022b049e461R543

Comment (juzhiyuan) [reply web/src/pages/Upstream/locales/zh-CN.ts:41]: or we could use `透传（pass）`

**Comment by juzhiyuan** [comment]
https://github.com/apache/apisix-dashboard/pull/1579/checks?check_run_id=2095317166

@nic-chen CI failed

**Review by juzhiyuan — APPROVED** [review]
LGTM, and welcome to add more descriptions for other fields in the future PRs.

**Comment by juzhiyuan** [comment]
BTW, because we may have long description 🤔 we could add Tooltip on an `info` icon, to have a better view.

**Review by LiteSun — APPROVED** [review]

**Comment by nic-chen** [comment]
> https://github.com/apache/apisix-dashboard/pull/1579/checks?check_run_id=2095317166
> 
> @nic-chen CI failed

OK. let's figure it out.

**Comment by moonming** [comment]
@guoqqqi @juzhiyuan You should achieve a better interaction, such as rewrite is related to the following selection, and the hint is not friendly

**Comment by juzhiyuan** [comment]
Sure, will do.

**Pull request merged**